app/services/data/get-approved-claim-expense-data.js

```diff
@@ -0,0 +1,45 @@
+const config = require('../../../knexfile').asyncworker
+const knex = require('knex')(config)
+
+module.exports.getClaimExpenseData = function(claimId) {
```

Comment:
Split into two simple data modules, each just a single function (e.g. `module.exports = function(claimId) {`). This makes them only have one concern and easier to understand how they are used in the worker and test.